<launch>

    <group ns="panda_arm">
        <node name="test_trajectory1" pkg="ros_control_boilerplate" type="test_trajectory" output="screen" >
            <param name="trajectory_controller" value="panda_1_joint_trajectory_controller" />
            <remap from="test_trajectory1/panda_1_joint_trajectory_controller/joints" to ="panda_1_joint_trajectory_controller/joints"/>
        </node>
        <node name="test_trajectory2" pkg="ros_control_boilerplate" type="test_trajectory" output="screen" >
            <param name="trajectory_controller" value="panda_2_joint_trajectory_controller" />
            <remap from="test_trajectory2/panda_2_joint_trajectory_controller/joints" to ="panda_2_joint_trajectory_controller/joints"/>
        </node>
    </group>

</launch>